network/runtime.go

```diff
@@ -0,0 +1,35 @@
+package network
+
+import "io"
+
+// These interfaces are intended to match those from whyrusleeping/cbor-gen, such that code generated from that
+// system is automatically usable here (but not mandatory).
+type CBORMarshaler interface {
+	MarshalCBOR(w io.Writer) error
+}
+
+type CBORUnmarshaler interface {
+	UnmarshalCBOR(r io.Reader) error
+}
+
+type CBORer interface {
+	CBORMarshaler
+	CBORUnmarshaler
+}
+
+// Specifies importance of message, LogLevel numbering is consistent with the uber-go/zap package.
+type LogLevel int
```

Comment:
Maybe a new `log` package? Maybe just introduce a runtime package?  To reduce imports, we can type alias any of these inside specs-actors anyways.

Reply:
`log` could be an annoying collision for consumers. `runtime` or `rt` could be ok, tho I would then wish that `ExitCode` was there too.

Reply:
Going with `rt`.  Will file an issue for moving exitcodes there as a nice to have if things are going really well early next week.